process-manager/process/src/test/java/org/orbisgis/orbisdata/processmanager/process/ProgressMonitorTest.java

```diff
@@ -0,0 +1,108 @@
+package org.orbisgis.orbisdata.processmanager.process;
+
+import org.junit.jupiter.api.Test;
+import org.orbisgis.orbisdata.processmanager.api.IProgressMonitor;
+
+import static java.lang.Math.round;
+import static org.junit.jupiter.api.Assertions.*;
+
+/**
+ * Test class dedicated to {@link ProgressMonitor} class.
+ *
+ * @author Erwan Bocher (CNRS)
+ * @author Sylvain PALOMINOS (Lab-STICC UBS 2020)
+ */
+class ProgressMonitorTest {
+
+    @Test
+    public void progressTest(){
```

Comment:
Would be nice to add an example within a IProcess

Reply:
Ok, I'll add it to the next PR which is about process.